design-documents/batch-query-services.md

```diff
@@ -0,0 +1,77 @@
+#Batch query services
```

Comment:
I think we need to add error reporting section as it's very important part for batch APIs.

Reply:
done